README.md

```diff
@@ -188,6 +188,25 @@ sudo xbps-install -S SDL2-devel SDL2_image-devel SDL2_net-devel alsa-lib-devel \
                      speexdsp-devel libatomic-devel libXi-devel
 ```
 
+``` shell
+# Nix 
+# With Home Manager on home.nix (Recommended Permanent Installation)
```

Comment:
Could you use normal sentence case for titles? (uppercasing important concept/entities as Home Manager is fine)